### PR TITLE
Dockerfile - Update pip install in docker images

### PR DESCRIPTION
**Pull request by yukirora**
**Description**
Update pip install in docker images to update the python package.

### CODE REVIEW

**Review by cp5555 — APPROVED** [review]

File: dockerfile/cuda11.1.1.dockerfile [hunk 1]
```diff
@@ -130,6 +130,7 @@ ADD third_party third_party
 RUN make -C third_party cuda
 
 ADD . .
-RUN python3 -m pip install .[nvworker] && \
+RUN python3 -m pip install --upgrade pip && \
+    python3 -m pip install -U .[nvworker] && \
```
Comment (abuccts) [inline dockerfile/cuda11.1.1.dockerfile:134]: why not fix the version for specific package instead? I think pandas has issue because they release a new version recently
upgrading to latest may have more issues for some other packages, like pytorch

Comment (cp5555) [reply dockerfile/cuda11.1.1.dockerfile:134]: When I read it, I also have the same feeling on this...

File: setup.py [hunk 2]
```diff
@@ -154,7 +154,7 @@ def run(self):
         'numpy>=1.19.2',
         'omegaconf==2.0.6',
         'openpyxl>=3.0.7',
-        'pandas>=1.1.5',
+        'pandas==1.1.5',
```
Comment (abuccts) [inline setup.py:157]: seems this line change is also merged in main?

File: .azure-pipelines/cuda-unit-test.yml [hunk 3]
```diff
@@ -11,7 +11,7 @@ pool:
 
 container:
   image: nvcr.io/nvidia/pytorch:20.12-py3
-  options: '-v /var/run/docker.sock:/var/run/docker.sock -v /usr/bin/docker:/usr/bin/docker'
+  options: "-v /var/run/docker.sock:/var/run/docker.sock -v /usr/bin/docker:/usr/bin/docker"
```
Comment (abuccts) [inline .azure-pipelines/cuda-unit-test.yml:14]: is this change necessary?

File: setup.py (hunk 2 above)
Comment (yukirora) [reply setup.py:157]: oh, if so, we don't need this pr
